code/modules/power/cable.dm

```diff
@@ -577,6 +577,13 @@ GLOBAL_LIST_INIT(wire_node_generating_types, typecacheof(list(/obj/structure/gri
 
 /obj/item/stack/cable_coil/proc/try_heal_loop(atom/interacting_with, mob/living/user, repeating = FALSE)
 	var/mob/living/carbon/human/attacked_humanoid = interacting_with
+	var/obj/item/clothing/under/uniform = attacked_humanoid.w_uniform
+	if(uniform && uniform.has_sensor == BROKEN_SENSORS)
+		visible_message(span_notice("[user] repairs the suit sensors on [interacting_with] with [src]."))
+		use(1)
+		uniform.has_sensor = HAS_SENSORS
+		return ITEM_INTERACT_SUCCESS
```

Comment:
This bit should be part of `/obj/item/clothing/under/item_interaction()`. Trying to use cable coils on others can call it, but the code itself belongs to the clothing code. This way players can also fix suits sensors by directly using cables on the uniform.

Reply:
Got it, I'll move it.

Reply:
faster reply in the coder west.

Reply:
Yeah you can already do it on uniform item. For on the mob I think it'd have to be either the coil like this, or on the mob/carbon/human's chain to check if the item is cable coil. Since cable coil already had a heal loop proc, that was my reasoning to put it in here.